src/hotspot/cpu/riscv/interp_masm_riscv_64.cpp

```diff
@@ -2364,7 +2355,7 @@ void InterpreterMacroAssembler::verify_oop_or_return_address(Register reg, Regis
   mr_PPC(R3_ARG1_PPC, R27_method);
   call_c(Rtmp); // call C
 
-  pop_frame();
+  pop_C_frame();
   restore_LR_CR(Rtmp);
```

Comment:
Shouldn't be these two lines `pop_C_frame(true/*restore_ra*/)`?

Reply:
Yeah, you're right